logstash-core/lib/logstash/environment.rb

```diff
@@ -29,6 +29,7 @@ module LogStash
             Setting::String.new("log.format", "plain", true, ["json", "plain"]),
             Setting::String.new("http.host", "127.0.0.1"),
               Setting::Port.new("http.port", 9600),
+            Setting::String.new("http.environment", "production"),
```

Comment:
why should we have this env here? talking about http env works for us (developers), but not really for end users, they should not care about it, and is also something that a release is always production, right?